extensions/yaml_support/setup.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+# Copyright 2004-present Facebook. All Rights Reserved.
+
+from setuptools import find_packages, setup
+
+setup(
+    name="yaml_support",
+    version="1.0.0",
+    description="A LabgraphUnits-YAML parser",
+    packages=find_packages(),
+    python_requires=">=3.6, <3.7",
```

Comment:
As LabGraph currently extends its support for python3.6+, can we relax this to >=3.6  ?

Reply:
Done